packages/irc-client/irc-client.0.3.2/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+authors: "John Else"
+homepage: "https://github.com/johnelse/ocaml-irc-client"
+bug-reports: "https://github.com/johnelse/ocaml-irc-client/issues"
+dev-repo: "git://github.com/johnelse/ocaml-irc-client"
+maintainer: "john.else@gmail.com"
+build: [
+  [make]
+]
+build-test: [
+  [make "doc"]
+  [make "test"]
+]
+install: [
+  [make "PREFIX=%{prefix}%" "install"]
+]
+remove: [
+  [make "PREFIX=%{prefix}%" "uninstall"]
+]
+depends: [
+  "oasis" {build}
+  "ounit"
+  "ocamlfind"
```

Comment:
You're right about ocamlfind - I've just pushed a fix.

ounit is unfortunately a build dependency since `make` builds the tests as well as the library.